src/main/java/org/sonarlint/cli/Main.java

```diff
@@ -132,6 +135,19 @@ public static void main(String[] args) {
       System.exit(ERROR);
     }
 
+    Charset charset = null;
+    try {
+      if (opts.charset() != null) {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) NullPointerException might be thrown as 'opts' is nullable here [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS2259)